minigotchi-ESP32/minigotchi.cpp

```diff
@@ -14,8 +15,34 @@
 
 // this code is pretty disgusting and shitty but it makes minigotchi.ino less cluttered!!!
 
+// current epoch val
+int Minigotchi::currentEpoch = 0;
+
+int Minigotchi::addEpoch() {
+    Minigotchi::currentEpoch++;
+    return Minigotchi::currentEpoch;
+}
+
+void Minigotchi::epoch() {
+    Minigotchi::addEpoch();
+    Serial.print("('-') Current Epoch: ");
+    Serial.println(Minigotchi::currentEpoch);
+    Serial.println(" ");
+}
+
+
 // things to do when starting up
 void Minigotchi::boot() {
+    // StickC Plus 1.1 and 2 power management, to keep turned On after unplug USB cable
```

Comment:
i think it would be better for the sake of the organization to have this done in `display.cpp`, maybe we can make a function called `Display::powerScreen();` that does this without it looking kinda messy in `minigotchi.cpp`. The handling/logic itself is fine however.

Reply:
Noted.